nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/PutHBaseRecord.java

```diff
@@ -75,6 +83,17 @@
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
+    protected static final PropertyDescriptor DEFAULT_VISIBILITY_STRING = new PropertyDescriptor.Builder()
```

Comment:
That's true using `pickVisibilityString` will add another complexity. However it can reduce the size of incoming Record formatted FlowFile.

My understanding is that currently each record needs to have the map field in order to use visibility labels even if there are 1000 records requiring the same visibility labels for cells in each record. In order to add such map field, user might have to do schema conversion and enrichment process before passing FlowFile to PutHBaseRecord processor. Having a smart (yet a bit complex necessarily) default mechanism would be helpful I think.